config/services/pay_by_link_payment/form.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return static function(ContainerConfigurator $container): void {
+    $container->import('form/**/*.php');
```

Comment:
quite a hussle just to load one form type. I find it strange that you've decided to extract form to separate folder, but kept context_provider and validator